docs/07-wsl-setup.md

```diff
@@ -0,0 +1,51 @@
+# How to run the webapp on linux subsystem on Windows (WSL1)?
+Most of the webapp as been developed from a Linux environment.
+To avoid environment specific troubleshooting and to improve performance it is recommended to set up the Windows Subsystem fot Linux (WSL) on Windows.
```

Comment:
```suggestion
To avoid environment specific troubleshooting and to improve performance it is recommended to set up and use the Windows Subsystem for Linux (WSL) on Windows.
```